src/examples/draggable/DropzoneDefault.tsx

```diff
@@ -0,0 +1,317 @@
+    message = 'Drag fruits here';
+  }
+
+  return (
+    <StyledDropzone ref={setNodeRef} isActive={isActive} isHighlighted={isHighlighted}>
+      {!isEmpty && (
+        <DraggableList>
+          {items.map(item => (
+            <DraggableList.Item key={item.id}>
+              <StyledDraggable {...item} />
+            </DraggableList.Item>
+          ))}
+        </DraggableList>
+      )}
+      {message && <Dropzone.Message>{message}</Dropzone.Message>}
+    </StyledDropzone>
+  );
+};
+
+const Example = () => {
+  const [columns, setColumns] = useState<IColumns>(defaultColumns);
+  const [activeId, setActiveId] = useState<string | null>(null);
+  const [dropzoneIsHighlighted, setDropzoneIsHighlighted] = useState<boolean>(false);
+
+  const theme = useTheme();
+  const overlayRef = useRef<HTMLDivElement>(null);
+
+  const activeItem = columns.draggables.find(item => item.id === activeId);
+  const draggableItems = columns.draggables;
+  const dropzoneItems = columns.dropzone;
+  const dropzoneItemCount = dropzoneItems.length;
+
+  const sensors = useSensors(
+    useSensor(MouseSensor),
+    useSensor(TouchSensor),
+    useSensor(KeyboardSensor, { coordinateGetter })
+  );
+
+  const onDragStart = useCallback(({ active }: DragStartEvent) => {
+    setActiveId(active.id as string);
+  }, []);
+
+  const onDragOver = useCallback(({ over }: DragOverEvent) => {
+    setDropzoneIsHighlighted((over?.id as string) === 'dropzone');
+  }, []);
+
+  const onDragEnd = useCallback(
+    ({ active, over }: DragEndEvent) => {
+      const overId = over?.id;
+
+      if (!overId) {
+        setActiveId(null);
+
+        return;
+      }
+
+      const activeColId = getColumnId(active.id as string, columns);
+      const overColId = getColumnId(overId as string, columns);
+
+      if (!overColId || !activeColId) return;
+
+      if (overId in columns) {
+        setColumns({
+          [activeColId]: columns[activeColId].filter(item => item.id !== active.id),
+          [overColId]: [...columns[overColId], activeItem!]
+        });
+      }
+
+      setActiveId(null);
+      setDropzoneIsHighlighted(false);
+    },
+    [columns, activeItem]
+  );
+
+  const onDragCancel = useCallback(() => {
+    setActiveId(null);
+    setDropzoneIsHighlighted(false);
+  }, []);
+
+  return (
+    <DndContext
+      accessibility={{ announcements: getAnnouncements(dropzoneItemCount) }}
+      sensors={sensors}
+      onDragStart={onDragStart}
+      onDragOver={onDragOver}
+      onDragEnd={onDragEnd}
+      onDragCancel={onDragCancel}
+      measuring={{ droppable: { strategy: MeasuringStrategy.Always } }}
+    >
+      <Row justifyContent="center">
+        <Col size={4}>
+          <StyledHeading isBold>List of produce</StyledHeading>
+          <DraggableList>
+            {draggableItems.map(item => (
+              <DraggableListItem key={item.id} {...item} />
+            ))}
+          </DraggableList>
+        </Col>
+        <Col size={5}>
```

Comment:
```suggestion
        <Col sm={5}>
```

...ditto re: `StyledCol` w media query